FinderCompass/src/main/java/atomicstryker/findercompass/common/DefaultConfigFilePrinter.java

```diff
@@ -78,7 +85,7 @@ public void writeDefaultFile(File fileToPrint)
             pw.println("NoEnderEyeNeedle");
             pw.println("//");
             pw.println("// this is Diamond = block id 56, with a needle color of {51,255,204}, it scans 15 blocks horizontally, 1 vertically, from 1-16 height, every second");
-            pw.println("diamond_ore:51:255:204:15:1:1:16:0");
+            pw.println("oreDiamond:51:255:204:15:1:1:16:0");
             pw.println("//");

```

Comment:
Just so i don't get it wrong - the old names still work without changes, right? I do not want the QQ of users whose configs no longer work